tcms/testcases/models.py

```diff
@@ -590,8 +592,7 @@ def unique_error_message(self, model_class, unique_check):
                           ('bug_id', 'case_run'))
         if unique_check in bug_id_uniques:
             return 'Bug %d exists in run %d already.' % (self.bug_id, self.case_run.pk)
-        else:
-            return super(Bug, self).unique_error_message(model_class, unique_check)
+        return super(Bug, self).unique_error_message(model_class, unique_check)
```

Comment:
In Python 3 you can call `super().some_other_method`, i.e. no need to pass the class name and self as parameters to super.